README.md

```diff
@@ -1,2 +1,138 @@
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+   - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+
+
+# 📖 Mobile First Project <a name="about-project"></a>
+
+This project "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. HTML
+2. CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSShttps://developer.mozilla.org/en-US/docs/Web/JavaScript/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add book list
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A Web browser.
+2. A Code editor
+3. A Version cotrol sytem(Gt-msc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run these command:
+
+``` git clone git@github.com:Katended/awesome-books.git```
+
+
+### Usage
+
+To run the project, do the following:
+
+Open index.html using a live server.
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 David Katende
+
+- GitHub: [@githubhandle](https://github.com/Katended)
+- LinkedIn: [LinkedIn](linkedin.com/in/katende-david-01627461)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment <a name="live-demo"></a>
+
+Please click [here](https://katended.github.io/awesome-books/index.html) to view a live demo
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add desktop version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]( https://katended.github.io/awesome-books/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give me a star ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+- **How soon are new features coming out?**
+
+  - We currently do not have scheduled release time
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Acknowledgements <a name="acknowledgements"></a>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://opensource.org/license/mit/) licensed.
+
```

Comment:
- [ ] The link to the MIT LICENSE that you provided is a template and not an actual LICENSE. Kindly provide a valid LICENSE. To do this, you can copy the text in the template into a file with the `.md` extension and then provide the link to that file here. 💻 😎 

![image](https://user-images.githubusercontent.com/77106962/236043922-5f4a3d79-03cf-4db2-bc66-554c5bb9cbbb.png)